### PR TITLE
add set-token option to add the auth token

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by alexrashed — CHANGES_REQUESTED** [review]
In general, I think it's a great idea to push the usage of the CLI and especially the `set-auth` command. However, it cannot really replace setting the environment variables when using other means to start LocalStack. I think it might be necessary to rewrite a bit more in the docs to make this distinction clear to the users, while using the full potential of the CLI?

File: content/en/user-guide/aws/dms/index.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ git clone https://github.com/localstack-samples/sample-dms-kinesis-rds-mariadb.g
 Next, start LocalStack (there is a docker-compose included, setting the `ENABLE_DMS=1` flag):
 
 ```sh
-export LOCALSTACK_AUTH_TOKEN=<your-auth-token> # this must be a enterprise license token
+localstack auth set-token <your-auth-token> # this must be a enterprise license token
```
Comment (alexrashed) [inline content/en/user-guide/aws/dms/index.md:40]: This one still uses the auth token for `docker-compose` which won't work.

**Review by alexrashed — CHANGES_REQUESTED** [review]

**Review by alexrashed — APPROVED** [review]
LGTM! This will make the `localstack auth set-token` command more prominent! 🚀

**Review by remotesynth — APPROVED** [review]
thanks for making this update!

**Pull request merged**